addons/editor-theme3/addon.json

```diff
@@ -324,9 +327,75 @@
   "enabledByDefault": false,
   "presets": [
     {
-      "name": "3.0 Tweaks",
+      "name": "High contrast",
+      "id": "contrast",
+      "description": "Scratch 3.0's upcoming block color palette shown in Scratch Lab",
+      "values": {
+        "motion-color": "#80B5FF",
+        "looks-color": "#CCB3FF",
+        "sounds-color": "#E19DE1",
+        "events-color": "#FFD966",
+        "control-color": "#FFBE4C",
+        "sensing-color": "#85C4E0",
+        "operators-color": "#7ECE7E",
+        "data-color": "#FFA54C",
+        "data-lists-color": "#FF9966",
+        "custom-color": "#FF99AA",
+        "Pen-color": "#13ECAF",
+        "sa-color": "#12EDD5",
```

Comment:
I think `#57DBD7` might be better - it's less saturated and closer to the default color for these blocks.